Add Mars Deepin 25 test reports and Ubuntu LTS reports. (#203)
ruyisdk/support-matrix: Mars/Deepin/README.md

### Operating System Information

- System Version: Deepin preview
- Download Link: <https://cdimage.deepin.com/RISC-V/preview-20240613-riscv64/deepin-23-beige-preview-riscv64-milkv-mars-20240613-123442.tar.xz>
- Reference Installation Document:
  1. <https://milkv.io/zh/docs/mars/getting-started/boot>
  2. <https://deepin-community.github.io/sig-deepin-ports/docs/install/riscv/jh7110>

### Hardware Information

- Milk-V Mars (8GB RAM)
- A USB power adapter and A USB-A to C or C to C cable
- A microSD card
- A USB to UART debugger (e.g., CH340, CH341, FT2232, etc.)
- A HDMI cable, A USB Mouse and A USB Keyboard

## Installation Steps

### Decompress and Flash Image to microSD Card

> - Merhod1: Use `tar` to decompress the image,  and then use `dd` to flash the image to the microSD card. (Assume/dev/sdc is the microSD card device)

Reviewer: ```suggestion
- Method 1: Use `tar` to decompress the image,  and then use `dd` to flash the image to the microSD card. (Assuming /dev/sdc is the microSD card device)
```

    ```bash

>     tar -xJvf deepin-25-beige-preview-riscv64-jh7110-20250122-110620.tar.xz

Reviewer: `-xvf` should be enough already.
```suggestion
    tar -xvf deepin-25-beige-preview-riscv64-jh7110-20250122-110620.tar.xz
```

    sudo dd if=deepin-jh7110-riscv64-25-desktop-installer.img of=/dev/sdc bs=4M status=progress
    sudo sync
    ```

- Method 2: After extracting the image, use the `balenaEtcher` software to write the image to the microSD card.

(The software may prompt verification failure when writing data. This does not affect the system startup)

> #### Issue: unabel to mount root file system

Reviewer: ```suggestion
#### Issue: unable to mount root file system
```

The image written by the 'dd' command may fail to mount the file system when Mars is started, and the log output from the serial port is similar to:

```log
Failed to mount /dev/mmcblk1p4 as root file system.
```

or

```log
Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(0,0)
```

A fix in the boot file is needed.

> Run sudo blkid on your machine, you shall see something like:

Reviewer: Better wrap commands with \`\`.
```suggestion
Run `sudo blkid` on your machine, you shall see something like:
```
